test/src/integration/templates/enableRepo.test.js

```diff
@@ -12,116 +12,100 @@ const chai = require('chai');
-            const res = await enableTemplateRepos(repoUrls);
-            res.should.have.status(207);
-            res.body.forEach(subResponse =>
-                subResponse.status.should.equal(200)
-            );
-        });
-        it(`those repos appear as enabled in list of template repos`, async function() {
-            const res = await getTemplateRepos();
-            res.should.have.status(200);
-            reposToTest.forEach(repo =>
-                res.body.should.deep.include({
-                    ...repo,
-                    enabled: true,
-                })
-            );
-        });
-        it(`batch disabling multiple template repos returns 207 and sub-status 200 for each subrequest`, async function() {
-            const repoUrls = reposToTest.map(repo => repo.url);
-            const res = await disableTemplateRepos(repoUrls);
-            res.should.have.status(207);
-            res.body[0].status.should.equal(200);
-        });
-        it(`those repos appear as disabled in list of template repos`, async function() {
-            const res = await getTemplateRepos();
-            res.should.have.status(200);
-            reposToTest.forEach(repo =>
-                res.body.should.deep.include({
-                    ...repo,
-                    enabled: false,
-                })
-            );
-        });
-        it(`templates from the disabled repos do not appear in list of enabled templates`, async function() {
-            const res = await getTemplates({ showEnabledOnly: true });
-            res.should.have.status(204);
-            // TODO: switch over to the below when we have 2 repos (so that 1 repo can still be enabled, so we return 200 rather than 204 No Content)
-            // res.should.have.status(200)
-            // res.body.should.not.include.deep.members(defaultTemplates);
+                res.should.have.status(207);
+                res.body.forEach(subResponse =>
+                    subResponse.status.should.equal(200)
+                );
+            });
+            it(`those repos are listed as DISabled`, async function() {
+                const disabledRepos = testRepos.map(repo => {
+                    return {
+                        ...repo,
+                        enabled: false,
+                    };
+                });
+
+                const res = await getTemplateRepos();
+
+                res.should.have.status(200);
+                res.body.should.have.deep.members(disabledRepos);
+            });
+            it(`those repos' templates DON'T appear in list of ENabled templates`, async function() {
+                const res = await getTemplates({ showEnabledOnly: true });
+                res.should.have.status(200);
+                res.body.should.not.have.deep.members(templatesFromTestRepos);
+            });
+
+            it(`batch ENabling repos ${testRepos.length} returns 207 and sub-status 200 for each subrequest`, async function() {
```

Comment:
readability change:    it "returns 207 with sub-status 200 for each sub-request whilst batch enabling n repos"